e2e_tests/src/e2e_conduit/scenarios/__init__.py

```diff
@@ -0,0 +1,22 @@
+from e2e_conduit.fixtures.plugin_fixture import PluginFixture
+
+
+class Scenario:
+    """Data class for conduit E2E test pipelines"""
+
+    def __init__(
+        self,
+        name,
+        importer: PluginFixture,
+        processors: list[PluginFixture],
+        exporter: PluginFixture,
+    ):
+        self.name = name
+        self.importer = importer
+        self.processors = processors
+        self.exporter = exporter
+        self.accumulated_config = {}
+        self.conduit_dir = ""
+
```

Comment:
```suggestion
from dataclasses import dataclass

@dataclass
class Scenario:
    """Data class for conduit E2E test pipelines"""
    
    name,
    importer: PluginFixture,
    processors: list[PluginFixture],
    exporter: PluginFixture,
    accumulated_config: dict = {}
    conduit_dir: str = ""

```
Take the above with a big grain of salt. But I couldn't help the temptation to suggest using [Python's dataclasses](https://realpython.com/python-data-classes/), especially because of your comment.

Reply:
Nice, I haven't used these before. I initially reached for NamedTuples, but they're immutable so they don't quite fit here. I'll see if dataclasses work